docs/sdk-api-spec.md

```
@@ -0,0 +1,614 @@
+/** Raised by {@link Agent.log}. */
+export class LogError extends SingularityError {
+  constructor(message: string, options?: { cause?: unknown }) {
+    super("LOG_FAILED", message, options);
+    this.name = "LogError";
+  }
+}
+```
+
+Recovery recipe for `ConfirmationTimeoutError` (illustrative):
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)